data.sql

```diff
@@ -26,3 +26,26 @@ UPDATE animals SET owner_id=3 WHERE name='Gabumon' OR name='Pikachu';
 UPDATE animals SET owner_id=4 WHERE name='Devimon' OR name='Plantmon';
 UPDATE animals SET owner_id=5 WHERE name='Charmander' OR name='Squirtle' OR name='Blossom';
 UPDATE animals SET owner_id=6 WHERE name='Angemon' OR name='Boarmon';
+
+INSERT INTO vets (NAME, AGE, DATE_OF_GRADUATION) VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (NAME, AGE, DATE_OF_GRADUATION) VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (NAME, AGE, DATE_OF_GRADUATION) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets (NAME, AGE, DATE_OF_GRADUATION) VALUES ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO SPECIALTIES(SPECIES_ID, VET_ID) VALUES (1, 2);
+INSERT INTO SPECIALTIES(SPECIES_ID, VET_ID) VALUES (2, 3);
+INSERT INTO SPECIALTIES(SPECIES_ID, VET_ID) VALUES (2, 4);
+
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (1, 4, '2021-02-24'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (2, 2, '2019-12-21'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (2, 1, '2020-08-10'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (2, 2, '2021-08-07'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (3, 3, '2019-09-29');
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (4, 4, '2020-10-03');
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (4, 4, '2020-11-04'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2019-01-24'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2019-05-15'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2020-02-27'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2020-08-03'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (6, 3, '2020-05-24'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (6, 1, '2021-01-11'); 
```

Comment:
- [ ] Good job working on this but please kindly include all missing data insert queries.